pkg/controller/.import-restrictions

```diff
@@ -242,6 +242,7 @@
         "k8s.io/kubernetes/pkg/scheduler/algorithm",
         "k8s.io/kubernetes/pkg/scheduler/algorithm/predicates",
         "k8s.io/kubernetes/pkg/scheduler/nodeinfo",
+        "k8s.io/kubernetes/pkg/scheduler/framework/plugins/helper",
```

Comment:
```"k8s.io/kubernetes/pkg/scheduler/algorithm"``` and ```"k8s.io/kubernetes/pkg/scheduler/algorithm/predicates"``` will be removed once https://github.com/kubernetes/kubernetes/issues/86717 is addressed.